aws_list_all/__main__.py

```diff
@@ -119,6 +137,47 @@ def main():
     )
     introspecters.add_parser('debug', description='Debug information', help='Debug information')
 
+    # Print the findings from query to an HTML file to be viewed in a browser
+    viewhtml = subparsers.add_parser(
+        'print-html', description='Print a listing to an HTML file to be viewed in a browser',
+        help='Create browser view of listing'
+    )
+    viewhtml.add_argument(
+        '-s',
+        '--service',
+        action='append',
+        help='Restrict querying to the given service (can be specified multiple times)'
+    )
+    viewhtml.add_argument(
+        '-r',
+        '--region',
+        action='append',
+        help='Restrict querying to the given region (can be specified multiple times)'
+    )
+    viewhtml.add_argument(
+        '-o',
+        '--operation',
+        action='append',
+        help='Restrict querying to the given operation (can be specified multiple times)'
+    )
+    viewhtml.add_argument(
+        '-u',
+        '--unfilter',
+        action='append',
+        help='Exclude given default-value filter from being applied (can be specified multiple times)'
+    )
+    viewhtml.add_argument('-p', '--parallel', default=32, type=int, help='Number of request to do in parallel')
+    viewhtml.add_argument('-d', '--directory', default='.', help='Directory to save result listings to')
+    viewhtml.add_argument('-v', '--verbose', action='count', help='Print detailed info during run')
+    viewhtml.add_argument('-c', '--profile', help='Use a specific .aws/credentials profile.')
```

Comment:
Shouldn't viewhtml be more similar to "show"? This looks like it's doing all the querying again.